network-api/networkapi/templates/fragments/buyersguide/related_product.html

```diff
@@ -0,0 +1,21 @@
+{% load localization wagtailimages_tags%}
+
+<div class="related-product">
+  <a class="tw-block {% if related_product.adult_content %} adult-content{% endif %}" href="{% relocalized_url related_product.url %}">
+    <div class="img-container tw-p-5">
+      {% image related_product.image width-600 as img %}
+      <img
+        class="product-thumbnail"
+        width="600"
+        loading="lazy"
+        src="{{img.url}}"
+        alt="{{related_product.title}}"
```

Comment:
**suggestion - non blocking**

I'm not privy to the `alt` system on the site, however, I think the images alt attribute should come from `related_product.image.alt` (if it's there) - but you might be intentionally using the title here down to publishing styles, just thought it worth mentioning.

Reply:
Good point. I have extracted this nearly as is from the product page. Looks like we are using the default Wagtail image model and don't have an alt field. The `alt` property would only return the image title and from what I can see on prod, this is not always fit for use as alt text (often just the filename). I guess using the product title might make more sense here, because all images I have seen do display the product in question (or a depiction there of -- e.g. logo of an app). I am actually wondering if we should just remove it, since it does not add any information..